record/record.app.js

```diff
@@ -46,7 +48,12 @@
         $rootScope.showDeleteButton = chaiseConfig.deleteRecord === true ? true : false;
 
         try {
-            var ermrestUri = UriUtils.chaiseURItoErmrestURI($window.location);
+            // TODO: Redirecting to home page is appropriate for an error in chaiseURItoErmrestURI
+            // try {
+                var ermrestUri = UriUtils.chaiseURItoErmrestURI($window.location);
+            // } catch (e) {
+            //     ErrorService.errorPopup(exception.message, exception.code, "home page");
+            // }
 
             context = $rootScope.context = UriUtils.parseURLFragment($window.location, context);
 
```

Comment:
instead of checking if context.filter exists, just do a verify assertion because it doesn't make sense to view Record app without a filter.